leaf-ui/css/style.css

```diff
@@ -140,16 +140,7 @@ body.waiting *
   color: #bcbcbc;
   text-shadow: 0 -1px 0 #000;
   background-color: rgba(90, 90, 90, 1);
-}
-
-#analysis-sidebar a{
-  padding: 16px;
-  display: block;
-}
-
-#analysis-sidebar a:hover {
```

Comment:
Why did you remove the hovering feature and would it be possible to bring it back?

Reply:
We initially removed it because it was making the configurations sidebar look rather odd. We will add it back in the next commit.